test/scale/test-scale.sh

```diff
@@ -41,6 +41,7 @@ OPTIONAL PARAMETERS:
     --debug-exit-after-print-counts       skip scale test. Just print out counts of things to be created and counts of IPSets/ACLs that NPM would create
     --num-real-services                   cluster ip service for the real deployments scheduled. Each svc will point to the respective deployment(having <num-real-replicas> pods) Default is 0
     --debug-exit-after-generation         skip scale test. Exit after generating templates
+    --num-real-nginx-deployments          create real-nginx-deployments, real-deployments should be set to 0 when this is in use. Default is 0
```

Comment:
Like the idea. Few thoughts:
1. Probably should prevent possibility of setting both nginx and agnhost deployments.
2. There are some existing dependencies in `numRealPods` and `numRealDeployments`.

May be cleaner to set something like `real-pod-type=agnhost|nginx`? Then `numRealPods` would be the number of nginx Pods, etc.

Reply:
Thanks, took this into account and made the modifications so that we keep numRealPods and numRealDeployments unchanged. Added realPodType to change the deployment types